packages/pigment-css-react/src/index.ts

```diff
@@ -8,3 +8,4 @@ export { default as createUseThemeProps } from './createUseThemeProps';
 export { default as internal_createExtendSxProp } from './createExtendSxProp';
 export { default as useTheme } from './useTheme';
 export { default as globalCss } from './globalCss';
+export { default as internal_styledEngineMockup } from './styledEngineMockup';
```

Comment:
This should not be part of the public api (base module export) but can come from a path like `@pigment-css/react/internal`.

Reply:
I followed the convention for `internal_createExtendSxProp` right above. I'll move it to a new internal folder.